packages/evm-adapter/src/events.ts

```diff
@@ -0,0 +1,21 @@
+export type PaymentEvent = {
+  sender: string;
+  recipient: string;
+  tokenAddress: string;
+  transferAmount: bigint;
+  feeAmount: bigint;
+};
+
+export type EthPaymentEvent = {
+  sender: string;
+  recipient: string;
+  transferAmount: bigint;
+  feeAmount: bigint;
+};
+
+export enum EventName {
+  Payment = 'Payment',
```

Comment:
please make the key upper case since its essentially constant

Reply:
OK